include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -917,7 +1050,7 @@ namespace VectorTools
       additional_data.mapping_update_flags = (update_values | update_JxW_values);
       MatrixFree<dim, Number> matrix_free;
       matrix_free.reinit (mapping, dof, constraints,
-                          QGauss<1>(fe_degree+1), additional_data);
+                          QGauss<1>(fe_degree+2), additional_data);
```

Comment:
I will create a separate PR which does what you want without changing the templates...